src/cmd.rs

```diff
@@ -28,22 +28,24 @@ mod stmsecure;
 mod tasks;
 mod test;
 mod trace;
+mod vsc7448;
 
 use crate::core::Core;
 use crate::hubris::*;
 use crate::Args;
 use crate::{attach_dump, attach_live};
-use anyhow::{bail, Result};
+use anyhow::{bail, Context, Result};
 use std::collections::HashMap;
 use std::convert::TryInto;
 use structopt::clap::App;
 
 #[allow(dead_code)]
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq)]
 pub enum Archive {
     Required,
     Optional,
     Prohibited,
+    Ignored,
```

Comment:
It feels like `Ignored` doesn't have materially different semantics as `Optional`?

Reply:
It's subtly different: `Optional` will load a Hubris archive if one is available (which is a 1-2 second startup penalty), vs `Ignored` which will not.

This means I can keep `HUMILITY_ARCHIVE` defined but still have the `info` subcommand be faster.

(It looks like `Prohibited` isn't actually used _anywhere_ – want me to take it out while I'm in here?)

Reply:
Ah, you use the environment variable! Yes, that makes sense.  It feels safe to ditch `Prohibited` -- and maybe worth adding a comment on each member as to the difference in semantics?

Reply:
Okay, done!